### PR TITLE
Create new readme and add LICENSE

### PR DESCRIPTION
**Pull request by swedchef**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,135 @@
...
+|--|--|--|
+|SMB_INPUTPATH|/input|The path where files are read from (local or SMB)[^1]|
+|SMB_OUTPUTPATH|/output|The path where files are delivered to (local or SMB)[^1]|
+|SMB_USERNAME|None|The username to use for reading/writing files in case SMB is used|
+|SMB_PASSWORD|None|The password to use for reading/writing files in case SMB is used|
+|ARCHIVE|FALSE|If not 'FALSE', archiving will be enabled (see [Archiving](#archiving))|
+|ARCHIVE_CLEAN_INTERVAL_H|24|Interval in hours between each archive-folder scan (check for old files)|
+|ARCHIVE_MAX_AGE_D|60|Maximum allowed age of archive-files (in days) before they are deleted|
+|SLEEPTIME|5|Time between each scan of the input-folder, and thereby file-move|
+|CLEAR_INPUT|FALSE|Port of the ksql-server (will be ignored if port is included in KSQL_HOST)|
```
Comment (stigsberg) [inline README.md:21]: Describtion to this variable CLEAR_INPUT seems to have a copy/paste error or at least the link between the two is understood by reading it.

**Review by stigsberg — CHANGES_REQUESTED** [review]
1 comment have to be fixed in order to have the variable explained more clearly.

**Comment by stigsberg** [comment]
In the system I have acces to I do not have sudo rights and therefore it have not been tried to spin up the container locally.

File: README.md (hunk 1 above)
Comment (swedchef) [inline README.md:21]:
```suggestion
|CLEAR_INPUT|FALSE|If not 'FALSE', files will be deleted after being read in 'input' folder|
```

**Review by vjsaini75 — APPROVED** [review]

**Review by vjsaini75 — APPROVED** [review]

**Review by stigsberg — APPROVED** [review]
LGTM

**Pull request merged**